Src/IronPython/Runtime/Operations/BigIntegerOps.cs

```diff
@@ -897,9 +932,9 @@ private static bool IsInt32(this BigInteger self)
             return __new__(context, type, val);
         }
 
-        #endregion
+#endregion
```

Comment:
Editor accidentally messing with the alignment? Otherwise seems fine to me, feel free to merge.